.github/workflows/goreportcard.yaml

```diff
@@ -0,0 +1,52 @@
+name: Go Report Card
+
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+
+jobs:
+  goreportcard:
+    strategy:
+      matrix:
+        go: ['stable']
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Setup Go ${{ matrix.go }}
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go }}
+        cache: false
+    - name: Checkout gojp/goreportcard repo
+      uses: actions/checkout@v4
+      with:
+        repository: gojp/goreportcard
+        path: goreportcard
+    - name: Install goreportcard-cli
+      # goreportcard-cli requires the following linters:
+      #  1. gometalinter
+      #  2. golint
+      #  3. gocyclo
+      #  4. ineffassign
+      #  5. misspell
+      # among which, the linter gometalinter is deprecated. However, goreportcard repo has a vendor version of it.
+      # Hence installing from the repo instead of `go install`. Refer https://github.com/gojp/goreportcard/issues/301
+      run: |
+        cd goreportcard
+
+        # Install prerequisite linter binaries: gometalinter, golint, gocyclo, ineffassign & misspell
+        # Refer: https://github.com/gojp/goreportcard?tab=readme-ov-file#command-line-interface
+        make install
+
+        # Install goreportcard-cli binary
+        go install ./cmd/goreportcard-cli
+        which goreportcard-cli
```

Comment:
Leftover from debugging?
```suggestion

```

Reply:
No, the idea was to fail fast when the binary is not found in the path or issues like that. But I think we can remove it.